### DISCUSSION TITLE
Autolab logo

### DISCUSSION THREAD

**Discussion by Python-simulation** [qcha41/usit#27] [Polls]
Autolab logo: ![autolab-icon](https://github.com/autolab-project/autolab/assets/46069237/16c7084f-bc4a-46cd-a19c-df50a54fd1de)

What it looks like in GUI:
![image](https://github.com/autolab-project/autolab/assets/46069237/81d83d10-0ef4-4b0c-b06c-ab27cf895970)

**Comment by Python-simulation** [comment]
@pyMatJ, @Totorminator98 opinion request :)

**Reply by pyMatJ** [reply]
Very nice! As long as it is not a 3dB bandwidth I am happy ;)

**Reply by Python-simulation** [reply]
Oh I didn't think about a 3dB bandwidth, don't give me ideas I might like :)
